API_DOCUMENTATION.md

```diff
@@ -28,27 +28,34 @@
   {
   text: string,
   senderId: int,
-  receiverId: int
+  receiverId: int,
+  respondsToId: int,
+  file: byte[],
+  fileName: String
   }
 
   **Response:**
   STATUS 200 OK
 
-#### GET "/messages/{userId1}/{userId2}"
+#### GET "/messages"
   *Get all messages sent between user1 and user2 in chronological order*
   
-  **Params:**
-  userId1: int
-  userId2: int
-
+   **Params:**
+  {
+  userId1: int,
+  userId2: int,
+  dateTimeFrom: Timestamp
+  }
   **Response:**
   [
     {
       id: int,
       text: string,
       time: timestamp,
-      sender: User object,
-      receiver: User object
+      senderId: int,
+      receiverId: int,
+      respondsToId: int,
+      hasFiles: Boolean
     }
   ]
 
```

Comment:
Update the documentation by describe the new "POST /messages" in detail. Details such as that the value under file key can be sent as integer array or base64 string.

Also describe the new "GET /messages/1/attachments